src/lib/tooltip/tooltip.ts

```diff
@@ -83,6 +84,23 @@ export const MD_TOOLTIP_SCROLL_STRATEGY_PROVIDER = {
   useFactory: MD_TOOLTIP_SCROLL_STRATEGY_PROVIDER_FACTORY
 };
 
+/**
+ * Interface used to register tooltip a11y message elements, with a count of how many tooltips have
+ * the message and the reference to the a11y message element used for the aria-describedby.
+ */
+export interface RegisteredA11yMessage {
+  element: HTMLElement;
+  count: number;
+}
+
+/** ID used for the body container where all a11y messages are appended. */
+export const A11Y_MESSAGES_CONTAINER_ID = 'md-tooltip-a11y-messages';
+
+/** Global incremental identifier for each registered a11y message. */
+let latestA11yMessageId = 0;
```

Comment:
I'd be okay with just calling this `nextId` (which is what we typically name these).